### PR TITLE
ref(upgrades): Remove errors_v2 and errors_v2_ro storages

### PR DESCRIPTION
**Pull request by nikhars**
Similar to https://github.com/getsentry/snuba/pull/3294, but this one is for errors_v2 and errors_v2_ro storages.

### CODE REVIEW

File: snuba/cli/offline_replacer.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 @click.option(
     "--storage",
     "storage_name",
-    type=click.Choice(["errors", "errors_v2"]),
+    type=click.Choice(["errors"]),
```
Comment (fpacifici) [inline snuba/cli/offline_replacer.py:24]: Should we just remove the parameter altogether ?

**Review by fpacifici — COMMENTED** [review]
Please fix the type error

File: snuba/cli/offline_replacer.py (hunk 1 above)
Comment (lynnagara) [reply snuba/cli/offline_replacer.py:24]: I think explicit is good. Offline replacer should work for others too, just a coincidence there is only one today.

Comment (nikhars) [reply snuba/cli/offline_replacer.py:24]: I would like to keep it for now. There is no harm in having it.

**Pull request merged**